src/app/pages/viewing/messages/messages.component.html

```diff
@@ -8,36 +8,59 @@
-        <input type="checkbox" class="custom-control-input" id="customSwitch1" [(ngModel)]="viewService.isCarouselView"
-               (click)="toggleViewStyle()">
-        <label class="custom-control-label toggle-switch" for="customSwitch1" *ngIf="!viewService.isCarouselView">Switch
-          to Booklet</label>
-        <label class="custom-control-label toggle-switch" for="customSwitch1" *ngIf="viewService.isCarouselView">Switch
-          to Cards</label>
-      </div>
+      <button class="btn-sm btn-primary" *ngIf="!viewService.isCarouselView" (click)="toggleViewStyle()">Switch to Booklet</button>
+      <button class="btn-sm btn-primary" *ngIf="viewService.isCarouselView" (click)="toggleViewStyle()">Switch to Cards</button>
     </div>
   </div>
 
   <!-- Card message view -->
   <div class="card-columns mt-3" *ngIf="viewService.isCarouselView === false">
-    <div class="card" *ngFor="let msg of viewService.messages;" id="{{ msg.id }}">
-      <div class="card-header">
-        <div class="card-header-author">
-                  <span class="profile-pic-container-small mr-2">
-          <img class="profile-pic" src="{{ msg.fromPic }}" alt="profile-pic">
-        </span>
-          <span class="card-header-text">{{ msg.fromName }}</span>
+    <div class="card shadow" *ngFor="let msg of viewService.messages;" id="{{ msg.id }}">
+      <div class="msg-header" [ngStyle]="{'background': generateHeaderColor(msg.backColor).bg, 'color': generateHeaderColor(msg.backColor).text}">
+        <div class="msg-header-group">
+          <table class="header-table">
+            <tr>
+              <td>
+              <span class="profile-pic-container-small mr-2">
+                <img class="profile-pic" src="{{ msg.fromPic }}" alt="profile-pic">
+              </span>
+              </td>
+              <td>
+                <table>
+                  <tr>
+                    <div class="msg-header-text">
+                      <span>{{ msg.fromName }}</span>
+                    </div>
+                  </tr>
+                  <tr>
+                    <span class="msg-header-date">{{ viewService.timeSince(viewService.nowMillis, msg.timestamp.toMillis(), msg.timestamp.toDate().toDateString()) }}</span>
+                  </tr>
+                </table>
+              </td>
+            </tr>
+          </table>
+        </div>
+        <div class="card-header-group">
+          <table class="header-table">
+            <tr>
+              <td>
+                <button class="btn card-button" type="button" data-toggle=""
+                        data-target="#bookmark">
+                  <img class="card-icon" src="../../../../assets/images/icons/bookmark.svg">
+                </button>
+                <button class="btn card-button" type="button" data-toggle="modal"
+                        data-target="#delete" (click)="setMessageToDelete(msg.id)">
+                  <img class="card-icon" src="../../../../assets/images/icons/trash.svg">
```

Comment:
Missing required alt attribute
```suggestion
                  <img class="card-icon" src="../../../../assets/images/icons/trash.svg" alt="Delete Message">
```